testsuite/layers/src/test/java/org/jboss/as/test/layers/LayersTestCase.java

```diff
@@ -31,6 +31,7 @@ public class LayersTestCase {
     // This is the expected set of un-referenced modules found when scanning
     // the test-standalone-reference configuration.
     private static final String[] NOT_REFERENCED = {
+        "javax.xml.stream.api",
```

Comment:
The usage of this module is unexpected. This change can be removed by removing the `javax.xml.stream.api` dependency from `org.wildfly.extension.elytron.jaas-realm`

Reply:
Removed in https://github.com/wildfly/wildfly-core/pull/5861